gufe/tests/test_serialization_json.py

```diff
@@ -229,6 +230,8 @@ def setup_method(self):
                 "path": "foo/bar",
             }
         ]
+        if sys.version_info[:2] >= (3, 13):
+            self.dcts[0]["__module__"] = "pathlib._local"
```

Comment:
This is roughly my understanding of what's going on here: https://github.com/sphinx-doc/sphinx/issues/13178

It would be worth getting more eyes on this, I _think_ it'll work, but I'm not sure if there's deeper implications, for example, what if you're deserializing something that was created with an older version of Python? Do we need to do some kind of migration within the JSON codec?

Reply:
yeah I'm concerned about interoperability with 3.12 and 3.13 serialization. i'll think on this a bit more - open to thoughts from @dotsdl

Reply:
trying out something here: https://github.com/OpenFreeEnergy/gufe/pull/577